core/src/main/java/org/nervos/ckb/utils/Witness.java

```diff
@@ -0,0 +1,22 @@
+package org.nervos.ckb.utils;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import org.nervos.ckb.crypto.secp256k1.ECKeyPair;
+import org.nervos.ckb.crypto.secp256k1.Sign;
+
+public class Witness {
```

Comment:
Won't it make more sense to have this inside `org.nervos.ckb.methods.type.Witness` or `org.nervos.ckb.methods.type.Transaction`?

Reply:
yes